proxy/src/telemetry/metrics/mod.rs

```diff
@@ -120,6 +127,12 @@ pub struct Aggregate {
 #[derive(Debug, Clone)]
 pub struct Serve {
     metrics: Arc<Mutex<Metrics>>,
+    handle: reactor::Handle,
+}
+
+#[derive(Clone, Debug)]
```

Comment:
Please document what a `WriteBody` is supposed to be. It is not clear what abstraction this is representing. In particular, let's document its memory allocation characteristics.